### ISSUE TITLE
Decrypt QR-codes

### ISSUE THREAD

**Issue by btclinux** [SafePalWallet/safepal-app#1]
How to decrypt QRs generted by software wallet and hardware S1, using this repo. At least some documentation is needed

**Comment by daic11** [comment]
Yes please, it's very important to the community to be more open source. Please decrypt QRs.
